### ISSUE TITLE
It gives errors in large strings.

### ISSUE THREAD

**Issue by hsdmr** [voku/anti-xss#80]
It gives errors in large strings. First, we got an error when trying large-size images in base64 format. Then we realized that if the size is roughly 1300 kb, it gives an error in plain texts as well.

**Comment by voku** [comment]
What is the error message?

**Comment by hsdmr** [comment]
We are not getting an error message. it deletes the string we sent directly. The isXssFound() method returns true.

**Comment by ampenidas** [comment]
hello, when trying to insert a base64 img in <div> and <img> tags, or a large string (bigger than ~300kb ) the library empties the content      

example:
```
<div><img src=\"{{base64img}}\" alt=\"\" width=\"1000\" height=\"555\" /></div>",
```

after cleaning
```
<div><img src=\"\" alt=\"\" width=\"1000\" height=\"555\" /></div>",
```

the same case happens on large strings       
example content is attached to message

thank you :)
[signature.txt](https://github.com/voku/anti-xss/files/7265114/signature.txt)

**Comment by voku** [comment]
Thanks for the error report, fixed in version 4.1.33
